src/core/lib/iomgr/ev_epoll1_linux.c

```diff
@@ -145,7 +145,7 @@ static const char *kick_state_string(kick_state_type st) {
 }
 
 struct grpc_pollset_worker {
```

Comment:
Why does this struct have the grpc_ prefix? (follow-on PR)

Reply:
There's a typedef in src/core/lib/iomgr/pollset.h